manuscript/chapter6-find-time.md

```diff
@@ -61,7 +63,18 @@ Another option is clearly assign tasks to someone else or delegating. This can b
 At the same time help you to avoid burnout, understand what is the amount of tasks that require you and others that can be done from others.  
 As contributor or leader is important to empower others to create better relationships but just do a better team job for the purpose of the project itself.  
 The benefits of the delegating can be very surprising because with empowering others there will be probably new opportunities to grow or change something that you never think of.  
-At the same time this avoid the gatekeeping that sometimes in OSS is a problem that block the project itself also for long time and creating a difficult feeling around the work that you are doing.
+At the same time this avoid the gatekeeping that sometimes in OSS is a problem that block the project itself also for long time and creating a difficult feeling around the work that you are doing.  
+
+## Burnout
+
+About this topics a lot of people with better knowledge then me wrote a lot of books.  
+My experience is basically to understand what are the priorities and define time slot, where after that you don't care except very important things (it is a skills understand what are them).  
+
+Usually burnout has different symptoms: isolation, feeling undervalued, recognition, values conflict, lose motivation and lack of autonomy.  
```

Comment:
*loss of motivation
or
*loosing motivation

Reply:
You could refer to https://fosdem.org/2020/schedule/event/burnout/ or https://selfcare.tech/ here